samples/highcharts/accessibility/accessible-pattern-fills/demo.html

```diff
@@ -0,0 +1,17 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/series-label.js"></script>
```

Comment:
We are not using series labels in this demo - remove the module and simplify the options? Can then remove all references to series.label.enabled = false in the options.